### DISCUSSION TITLE
Context menu should not remove focus of webview.

### DISCUSSION THREAD

**Discussion by cweijan** [microsoft/vscode-discussions#155] [Extension Development QnA] [answered]
When the context menu appears, the focus inside the webview will be removed, which causes this context The menu can only achieve limited features.

1. Open webview and giving the element the focus:
![image](https://user-images.githubusercontent.com/27798227/194452042-72145109-3dc0-4e49-a7a4-dd3b6ae3a46d.png)
2. After context menu appears, focus inside the webview will be removed.
![image](https://user-images.githubusercontent.com/27798227/194452087-393faaa6-f405-460c-806c-ff9a365fa2db.png)

**Comment by digitarald** [comment, marked as answer]
As this seems a concrete issues with reproducible steps, please file it at https://github.com/microsoft/vscode/issues/new/choose .
